tariff/octopus/graphql/api.go

```diff
@@ -111,7 +108,7 @@ func (c *OctopusGraphQLClient) AccountNumber() (string, error) {
 		return "", errors.New("no account associated with given octopus api key")
 	}
 	if len(q.Viewer.Accounts) > 1 {
-		c.log.WARN.Print("more than one octopus account on this api key - picking the first one. please file an issue!")
+		return "", errors.New("more than one octopus account on this api key not supported")
```

Comment:
my only proviso on this wording is that it implies we will never support it / people should give up with the integration, whereas we want people to report it so that account selection functionality can be added if necessary - happy to go with breaking loudly though